step-grid-parent/step-grid-agent/src/main/java/step/grid/agent/AgentServices.java

```diff
@@ -280,6 +286,19 @@ public void run() {
 		}.start();;
 	}
 
+	@GET
+	@Consumes(MediaType.APPLICATION_JSON)
+	@Path("/preStop")

```

Comment:
@david-stephan The standard best practice for REST APIs is to have a hyphen, not camelcase or underscores

Reply:
Changed endpoint path to pre-stop